test/string-checker.js

```diff
@@ -356,4 +356,24 @@ describe('modules/string-checker', function() {
             });
         }
     });
+
+    describe('getRegisteredPresets', function() {
+        it('returns the configuration\'s registered presets', function() {
```

Comment:
This test should actually check that it returns registered presets. The fact that it's a middleman is implementation detail.